### DISCUSSION TITLE
Vite: The requested module '...' does not provide an export named 'default'

### DISCUSSION THREAD

**Discussion by asudevprj** [quasarframework/quasar#14974] [General - Components / Directives / etc]
I am using VITE to develop a Capacitor mobile app. 
Several Capacitor plugins are used (sqlite, geolocation) which work fine.
Unfortunately, all BLE plugins (Capacitor or Cordova) I used in earlier projects (WEBPACK) do give an in the VITE setting an error:

Example for capacitor community plugin bluetooth-le:

SyntaxError: The requested module '/src-capacitor/node_modules/throat/index.js?v=4cc24925' does not provide an export named 'default'

The error happens in web dev mode and in android mode.
For testing, I created again a webpack project. There, the plugins work.

I would very much like to use Vite. Any help is very much appreciated!

**Comment by metalsadman** [comment]
try updating your quasar deps, `quasar upgrade -i`. also post your `quasar info` logs here.

**Reply by asudevprj** [reply]
Thank you for your suggestions!
As it shows, the quasar deps are up to date:
Operating System - Windows_NT(10.0.19044) - win32/x64
NodeJs - 16.13.2

Global packages
  NPM - 8.19.2
  yarn - 1.22.15
  @quasar/cli - 1.3.2
  @quasar/icongenie - Not installed
  cordova - 11.0.0

Important local packages
  quasar - 2.10.2 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-webpack - 3.6.2 -- Quasar Framework App CLI with Webpack
  @quasar/extras - 1.15.6 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.45 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.1.6
  pinia - Not installed
  vuex - Not installed
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  @babel/core - 7.20.2 -- Babel compiler core.
  webpack - 5.75.0 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 4.9.3 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  typescript - 4.5.5 -- TypeScript is a language for application scale JavaScript development
  @capacitor/core - 4.5.0 -- Capacitor: Cross-platform apps with JavaScript and the web
  @capacitor/cli - 4.5.0 -- Capacitor: Cross-platform apps with JavaScript and the web
  @capacitor/android - 4.5.0 -- Capacitor: Cross-platform apps with JavaScript and the web
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*


In the meantime I have set up my project under webpack, where all works just as expected. Therefore, the problem is with vite. It would be great if quasar could be used with vite and also, if quasar would create the actual Capacitor version by default.
